Cython/Plex/Transitions.py

```diff
@@ -50,10 +49,8 @@ def __init__(self, map=None, special=None):
             special = {}
         self.map = map
         self.special = special
-        #self.check() ###
 
-    def add(self, event, new_state,
-            TupleType=tuple):
+    def add(self, event, new_state, TupleType=tuple):
```

Comment:
I guess that the intention of the line breaks here was to separate the actual arguments from the cached globals. That makes perfect sense to me, so please keep it.

Reply:
this one feels really awkward to me, but OK

Reply:
Yeah… These days, one probably woudn't do this at all any more, and if I did, I'd at least prefix the names with underscores.